### PR TITLE
Add Docker support

### PR DESCRIPTION
**Pull request by fractalpenguin**

### CODE REVIEW

File: bot/cogs/fun.py [hunk 1]
```diff
@@ -98,7 +98,7 @@ def __init__(self, bot: Bot):
             await ctx.send(f"Unknown emojis: {emoji_string}")
 
     @command()
-    async def xkcd(self, ctx: Context, number=None):
+    async def xkcd(self, ctx: Context, number: str = None):
```
Comment (shadycake) [inline bot/cogs/fun.py]: Why is a number a string? Also kwargs shouldn't have spaces around the `=` if i recall correctly.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+FROM python:3
+
```
Comment (shadycake) [inline Dockerfile]: Add these lines to use tini
```
RUN apk add --update tini

ENTRYPOINT ["/sbin/tini", "--"]
```
Using this makes everything work more gracefully when shutting down.

**Comment by thebeanogamer** [comment]
@shadycake If the requested changes were made, would this be good to merge?

**Comment by thebeanogamer** [comment]
Also, as the CI files are only on the master branch, I can't run CI checks on this.

**Pull request merged**